app/views/children/_form.html.arb

```diff
@@ -39,12 +39,19 @@ context.instance_exec do
       end
 
       render 'children/form/childcare', form: self
+      if child.age <= 5
```

Comment:
What do you think about extracting this condition into a helper method with a descriptive name to help people understand the business rule behind this? Also, it would be nice to have the age limit of 5 defined in just one place (probably in a constant with a descriptive name).

Reply:
Helper now made :)